pkgs/tools/security/doas/default.nix

```diff
@@ -24,7 +25,7 @@ stdenv.mkDerivation rec {
   configureFlags = [
     (lib.optionalString withTimestamp "--with-timestamp") # to allow the "persist" setting
     "--pamdir=${placeholder "out"}/etc/pam.d"
-  ];
+  ] ++ lib.optional (!enablePAM) "--without-pam";
```

Comment:
I'd prefer this went inside the `configureFlags` list, just as `withTimestamp` above.